src/frontend/screens/MapScreen/index.tsx

```diff
@@ -39,6 +40,14 @@ export const MapScreen = () => {
   const {location} = useLocation({maxDistanceInterval: MIN_DISPLACEMENT});
   const savedLocation = useLastKnownLocation();
   const coords = location && getCoords(location);
+  const locationEnabled = useLocationProviderStatus();
+
+  if (
+    locationEnabled !== undefined &&
+    !locationEnabled.locationServicesEnabled &&
+    following
+  )
+    setFollowing(false);
```

Comment:
So the following could happen which would lead to jumpy behaviour in the map:

1. user turns `following` on
2. location service is disabled
3. if were to use your logic, there would be no fixed center coordinate (which is what want).
4. the user moves the map around manually.
5. Location service is enabled again.
6. since following is still set to true, it will jump the map back to users current location.

Reply:
You know what, actually I'm wrong. If user does action #4, `following` is [setToFalse](https://github.com/digidem/CoMapeo-mobile/blob/72850c81ebaecb2973651b0b8a35d243a3d352bb/src/frontend/screens/MapScreen/index.tsx#L85).

And if they don't do #4 than it will center screen to their newest location once location service is enabled again, which is what we want

Reply:
addhttps://github.com/digidem/CoMapeo-mobile/pull/136/commits/eda9e2a979883ee49846df1e08b5f341f9d30237